### DISCUSSION TITLE
`Build ipa for distribution` step exits with status code 65

### DISCUSSION THREAD

**Discussion by MbuthiaWaKihara** [codemagic-ci-cd/codemagic-docs#2840] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

Attached the error log file
[GroupAppMobile 4.log](https://github.com/user-attachments/files/17812738/GroupAppMobile.4.log)

### Expected results

Successful build

### Actual results

Build fails, Building ipa for distribution step fails

### Build id (optional)

673c54a46f847fca4a33c9c0

**Comment by dam-ease** [comment]
Hello @MbuthiaWaKihara,
```
error: unsupported option '-G' for target 'arm64-apple-ios12.4' (in target 'BoringSSL-GRPC' from project 'Pods')

Build failed :|
Step 7 script `Build ipa for distribution` exited with status code 65
```
I did a quick google search of this error and I see similar one with a suggested solution in [this](https://stackoverflow.com/questions/78608693/boringssl-grpc-unsupported-option-g-for-target-arm64-apple-ios15-0) SO thread, can you please check and try this out?

**Reply by MbuthiaWaKihara** [reply]
Hello @dam-ease 

Thank you for the suggestion. I tried it out. I think it solved the issue related to 'BoringSSL-GRPC', but I am still getting another error:

Build id: 673c8afe3581a3464ec6b20d

I have attached the error log file. Kindly help. Thanks!
[GroupAppMobile 6.log](https://github.com/user-attachments/files/17815562/GroupAppMobile.6.log)

**Reply by MbuthiaWaKihara** [reply, marked as answer]
I found a solution here: [37748](https://github.com/facebook/react-native/issues/37748)
